src/style.css

```diff
@@ -0,0 +1,94 @@
+/* #region fonts */
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 600;
+}
+/* #endregion */
+/* #region root */
```

Comment:
These comments are actually redundant here as the file is quite small. You can remove them